PlayCover/Views/AppSettingsView.swift

```diff
@@ -272,11 +271,8 @@ struct GraphicsView: View {
                                 })
                                 .frame(width: 125)
                         }
-                        onIncrement: {
```

Comment:
Is removing these new lines to ensure that the file is within the file length limit? If so, I think we should either add a SwiftLint exception for the file or find a way to split this into multiple files.

Reply:
> I think the best thing is to increase the length limit and modify the rules, 250 is too low

And yes is to avoid the limit.